MatrixKit/Controllers/MXKRoomViewController.m

```diff
@@ -3827,6 +3829,34 @@ - (void)showAttachmentInCell:(UITableViewCell*)cell
     }
 }
 
+- (BOOL)canPreviewFileAttachment:(MXKAttachment *)attachment withLocalFileURL:(NSURL *)localFileURL
+{
+    // Sanity check.
+    if (![NSFileManager.defaultManager isReadableFileAtPath:localFileURL.path])
+    {
+        return NO;
+    }
+    
+    if (UIDevice.currentDevice.systemVersion.floatValue >= 13)
+    {
+        return YES;
+    }
+    
+    MXKUTI *fileUTI = [[MXKUTI alloc] initWithLocalFileURL:localFileURL];
+    if (!fileUTI)
+    {
+        return NO;
+    }
+    
+    if ([attachment conformsToUTI:MXKUTI.html] || [attachment conformsToUTI:MXKUTI.xml] || [attachment conformsToUTI:MXKUTI.svg]
```

Comment:
We probably need a `conformsToUTIs` method and create a constant for supported UTI for preview in this class or in the BuildSettings